Sources/TuistKit/Commands/GraphCommand.swift

```diff
@@ -72,6 +78,7 @@ struct GraphCommand: AsyncParsableCommand, HasTrackableParameters {
             layoutAlgorithm: layoutAlgorithm,
             skipTestTargets: skipTestTargets,
             skipExternalDependencies: skipExternalDependencies,
+            noOpen: noOpen,
```

Comment:
I would invert the boolean when passing here so we don't have to write inverted logic
```suggestion
            open: !noOpen,
```